tests/snapshots/snap_test_holidata/test_holidata_produces_holidays_for_locale_and_year[de-DE-2011] 1.py

```diff
@@ -135,14 +135,6 @@
         'region': 'SL',
```

Comment:
Those are the wrong test files... 😅 You need to add those for `en_SG`.

Add your locale to [holidata/holidays/__init__.py](https://github.com/GothenburgBitFactory/holidata/blob/4bff11a0249fac9851637a4128e0a2639a79d020/src/holidata/holidays/__init__.py#L33), sorted alphabetically into the list, rerun `pytest tests/test_holidata.py --update-snapshots` to create the snapshots for SG, and add those (and only those). If any other tests are red or have to be modified, that's an error 😉